src/renderer/bootstrap.tsx

```diff
@@ -19,6 +19,17 @@ import { App } from "./components/app";
 import { LensApp } from "./lens-app";
 import { themeStore } from "./theme.store";
 
+/**
+ * If this is a development buid, wait a second to attach
+ * Chrome Debugger to renderer process
+ * https://stackoverflow.com/questions/52844870/debugging-electron-renderer-process-with-vscode
+ */
+async function attachChromeDebugger() {
+  if (isDevelopment) {
+    await delay(1000);
```

Comment:
need to import `delay`...

Reply:
Then I see this when running `yarn dev:renderer`:
```
    ERROR in ./src/renderer/bootstrap.tsx 71:0
    Module parse failed: Cannot use keyword 'await' outside an async function (71:0)
    File was processed with these loaders:
     * ./node_modules/@pmmmwh/react-refresh-webpack-plugin/loader/index.js
     * ./node_modules/ts-loader/index.js
    You may need an additional loader to handle the result of these loaders.
    | // run
    | bootstrap(process.isMainFrame ? LensApp : App);
    > await attachChromeDebugger();
    | 
    | 
     @ multi ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ReactRefreshEntry.js ./node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js ./src/renderer/bootstrap.tsx LensDev[2]
```

Reply:
Oh that is a different problem. That has to do with the `await attachChromeDebugger()` being called at the end of this file.

Reply:
Yeah, noticed that - already fixed